guides/doc-Quickstart_Guide/master.adoc

```diff
@@ -0,0 +1,86 @@
+include::common/attributes.adoc[]
+
+:imagesdir: common/images
+:numbered:
+ifdef::foreman-el,foreman-deb,katello[]
+:toc:
+:toc-placement: left
+endif::[]
+
+:quickstart-installation-guide:
+:context: {project-context}
+:mode: connected
+:ProductName: {ProjectServer}
+
+= {project-installation-guide-title}
+
+The Foreman installer is a collection of Puppet modules that installs everything required for a full working Foreman setup. It uses native OS packaging (e.g. RPM and .deb packages) and adds necessary configuration for the complete installation.
+
+Components include the Foreman web UI, Smart Proxy, a Puppet server, and optionally Passenger, TFTP, DNS and DHCP servers. It is configurable and the Puppet modules can be read or run in “no-op” mode to see what changes it will make.
+
+.Operating Systems supported by {foreman-installer}
```

Comment:
Rather than paste this, could we not reuse the adoc file for supported browsers? This would allow us to make changes once.

Reply:
Did you mean browsers here? Or operating systems? I can see about converting the operating systems table into a fragment that can be included.

Reply:
I meant operating system. Sorry, I am moving into my evening and obvs need more coffee :)

Reply:
@ehelms why would it have to be a fragment, if you do include::<filename> it will appear and you can adjust the offset. There isn't a lot in that file that's different from this.

Reply:
Which file are you thinking of including?

Reply:
I think I also need more coffee. I see exactly what you mean.